src/Libraries/Microsoft.Extensions.Http.Resilience/Hedging/ResilienceHttpClientBuilderExtensions.Hedging.cs

```diff
@@ -142,6 +143,11 @@ public static IStandardHedgingHandlerBuilder AddStandardHedgingHandler(this IHtt
         return new StandardHedgingHandlerBuilder(builder.Name, builder.Services, routingBuilder);
     }
 
+    [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+        Justification = "The EmptyResilienceStrategyOptions doesn't have any properties to validate.")]
+    private static ResiliencePipelineBuilder<HttpResponseMessage> AddStrategy(this ResiliencePipelineBuilder<HttpResponseMessage> builder, Func<StrategyBuilderContext, ResilienceStrategy> factory) =>
```

Comment:
@martintmk - do you think Polly could expose this API instead? That way callers wouldn't have to define an empty `ResilienceStrategyOptions` class and suppress the warning.

Reply:
I think we could. Could you create an issue in Polly repo?